metadata.json

```diff
@@ -57,6 +57,7 @@
         "6",
         "7",
         "8"
```

Comment:
You forgot a comma that I included in https://github.com/puppetlabs/puppetlabs-mysql/pull/966.